src/openrct2/drawing/X8DrawingEngine.cpp

```diff
@@ -133,8 +133,10 @@ void X8RainDrawer::Restore()
     }
 }
 
+#ifdef __WARN_SUGGEST_FINAL_TYPES__
```

Comment:
Wuah! Yes, indeed, sorry, thanks for reviewing carefully. Will re-push in a minute.